src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -644,6 +644,10 @@ bool ResumeCtrlImpl::RestoreApplicationData(ApplicationSharedPtr application,
   bool result = resumption_storage_->GetSavedApplication(
       application->policy_app_id(), device_mac, saved_app);
   if (result) {
+    if (saved_app.keyExists(strings::user_location)) {
+      application->set_user_location(saved_app[strings::user_location]);
+    }
+
```

Comment:
I think this should be in `ResumptionDataProcessorImpl::Restore`

Reply:
@iCollin fixed in https://github.com/smartdevicelink/sdl_core/pull/3622/commits/7d1f02a59c5cec1963918b5476ae9ff9ec6cd1b8